docs/source/2020EasterStorm.rst

```diff
@@ -99,6 +124,25 @@ Synoptic Dynamics
 
     * GFS.v16.0.10 generates a progressive synoptic pattern compared GFS_ANL. 
 
+  .. group-tab:: SRW.v1.0
+
+    .. figure:: images/2020Easter/MSLP_SRW_v1.0_2020EasterStorm_trim.png
+      :width: 1200
+      :align: center
+
+      Mean sea level pressure (hPa)
+
+    * The surface low shifts to the southeast side of the actual position in both SRW_RRFSv1alpha and SRW_GFSv15p2 forecasts.
+
+    .. figure:: images/2020Easter/500mb_SRW_v1.0_2020EasterStorm_trim.png
+      :width: 1200
+      :align: center
+
+      500 hPa geopotential heights (dam) and absolute vorticity (10 :sup:`-5`/s)
+
+    * Both SRW_GFSv15p2 and SRW_RRFSv1apha forecast the upper level trouh sitting at the downstream of the acutal position.
```

Comment:
Several typos and grammar: "forecast the upper level trough downstream of the actual position"

Reply:
Changed, thanks!